src/main/java/frc/robot/subsystems/Claw.java

```diff
@@ -0,0 +1,35 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+package frc.robot.subsystems;
+
+import edu.wpi.first.wpilibj.Compressor;
+import edu.wpi.first.wpilibj.DoubleSolenoid;
+import edu.wpi.first.wpilibj.PneumaticsModuleType;
+import edu.wpi.first.wpilibj.DoubleSolenoid.Value;
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+import frc.robot.Constants;
+
+public class Claw extends SubsystemBase {
+	/** Creates a new Claw. */
+
+	private Compressor m_phCompressor = new Compressor(Constants.PNEUMATIC_HUB_PORT, PneumaticsModuleType.REVPH);
+	private DoubleSolenoid m_doubleSolenoid = new DoubleSolenoid(Constants.DOUBLE_SOLENOID_MODULE_NUMBER, PneumaticsModuleType.REVPH, Constants.DOUBLE_SOLENOID_FORWARD_CHANNEL, Constants.DOUBLE_SOLENOID_REVERSE_CHANNEL);
+
+	public Claw() {
+		m_doubleSolenoid.set(Value.kReverse);
```

Comment:
Do we care about setting the claw on initialize? This is actually a little dangerous; the claw might suddenly open when the robot is still initializing.